### ISSUE TITLE
User would like to compare the results of two runs.

### ISSUE THREAD

**Issue by jterstriep** [openskope/SKOPE#2]
@Zodiase commented on [Mon Jul 31 2017](https://github.com/openskope/skope-interface/issues/54)

This is a **`User Story`**. It needs to be expanded with context and detail. Create separate PRs for feature implementations that support this User Story. The goal here is to develop and finalize the story of how the user would accomplish the task.

**Comment by abrin** [comment]
can this be done by opening two runs in separate side-by-side windows?

**Comment by jterstriep** [comment]
Yes, I think so. I'm going to try prototype something for review.

**Comment by rushgeo** [comment]
I think the comparison happens between two layers in the workspace. That suggests:

1. We need a good method of getting new layers into the workspace (a good search popup, or a way to return to the workspace after doing a search if that's a separate page)  
2. A way of starting the comparison  
3. A way of selecting the layers to compare  

For starting the comparison, perhaps clicking on one of the layers in the table of contents could provide "Compare" as an option.  
For selecting layers to compare, perhaps the UI could suggest layers based on the first selected layer. That could be layers of the same data type, e.g. both temperature layers.

**Comment by rushgeo** [comment]
#9 would provide "a good method of getting new layers into the workspace." That's should be the first step before we can compare them in any way (whether that's side-by-side, subtraction, etc.).

**Comment by abrin** [comment]
Do we need to recreate a GIS app by putting multiple layers in the workspace? This seems like a lot of work

**Comment by rushgeo** [comment]
Since we would still need a way to find the second layer to compare it side-by-side, I would expect that to be just as much work.

Side-by-side may be useful, but since it takes up more screen space than overlay, I'd be reluctant to make that the primary method of comparison. With overlays, the interface may be more usable on a tablet, although I don't know if that is a required platform.

We'll need some sort of multiple layer support, even if you can only view one at a time, since sometimes the search result isn't a single layer, it's a set of layers (e.g. all of the outputs created by PaleoCAR).

**Comment by kintigh** [comment]
I think we should just have limited capabilities here.  Visually switching between the two overlays for a given year, and maybe looking at differences would seem to be the main ones to me.  In a time series, that we are usually dealing with, a subtraction would give you a new layer.  Substraction could either be direct for the same kind of data or maybe be able to convert each layer to Z scores or percentiles (reporting the mean and SD) and then substract the Zscores.  You'd want to be able to graph the subtracted layers.

**Comment by abrin** [comment]
A question here about timing / prioritization -- this seems like a v2 type feature.  First get the system working with displaying one result and building out the features needed, and then expand to a more full-blown workspace if needed.  Am I seeing this right?

Also, is this a dup of issue #9 ?
